cvmfs/options.h

```diff
@@ -162,7 +183,9 @@ class OptionsManager {
  *  "KEY=VALUE".
  *
  *  No comments (#) are allowed.
+ *  No templating variables (@foo@) are allowed.
```

Comment:
This comment is not (anymore) accurate, is it?

Reply:
Yes, this was meant to be removed already...